src/operations/graph/graphHelpers.js

```diff
@@ -319,7 +319,8 @@ class GraphHelper {
                         .includes(`${relatedResource.resourceType}/${relatedResource.id}`)));
 
                     if (matchingParentEntities.length === 0) {
-                        throw new Error(`Forward Reference: No match found for child entity ${relatedResource.resourceType}/${relatedResource.id}` + ' in parent entities' + ` ${parentEntities.map(p => `${p.resource.resourceType}/${p.resource.id}`).toString()}` + ` using property ${property}`);
+                        const parentEntitiesString = parentEntities.map(p => `${p.resource.resourceType}/${p.resource.id}`).toString();
+                        throw new Error(`Forward Reference: No match found for child entity ${relatedResource.resourceType}/${relatedResource.id}` + ' in parent entities' + ` ${parentEntitiesString} using property ${property}`);
```

Comment:
1. Line too long.
2. If we are keeping the long line, any reason why this string is not made using a single templated string?

Reply:
Don't think line too long is being followed in the repo yet since there are too many instances of it. Updated the line to single string.